Description: Don't hardcode pkg-config to fix cross building
Author: Dennis Braun <d_braun@kabelmail.de>
Forwarded: yes

Index: lv2file/Makefile
===================================================================
--- lv2file.orig/Makefile
+++ lv2file/Makefile
@@ -1,6 +1,8 @@
+PKG_CONFIG ?= pkg-config
+
 CFLAGS += $(CPPFLAGS)
-CFLAGS += -O3 -Wall -Wextra --std=c99 `pkg-config --cflags argtable2 sndfile lilv-0`
-LDLIBS = `pkg-config --libs argtable2 sndfile lilv-0` -lm
+CFLAGS += -O3 -Wall -Wextra --std=c99 `$(PKG_CONFIG)  --cflags argtable2 sndfile lilv-0`
+LDLIBS = `$(PKG-CONFIG)  --libs argtable2 sndfile lilv-0` -lm
 BINDIR = $(DESTDIR)/usr/bin
 INSTALL_PROGRAM = install
 
